Resources/public/js/views/ez-locationviewview.js

```diff
@@ -346,6 +346,14 @@ YUI.add('ez-locationviewview', function (Y) {
                             selected: false,
                             bubbleTargets: this,
                         }),
+                        new Y.eZ.LocationViewLocationsTabView({
+                            content: this.get('content'),
+                            location: this.get('location'),
+                            config: this.get('config'),
+                            priority: 3000,
```

Comment:
As "Locations" tab should be before "Content relations" to keep consistency of priority I have changed priority for your tab.